contrib/mesos/pkg/minion/server.go

```diff
@@ -132,6 +153,11 @@ func (ms *MinionServer) launchExecutorServer() {
 	ms.AddExecutorFlags(executorFlags)
 	executorArgs, _ := filterArgsByFlagSet(allArgs, executorFlags)
 
+	executorArgs = append(executorArgs, "--resource-container="+path.Join("/", ms.cgroupRoot, "kubelet"))
```

Comment:
my intent was specifically to provide `--resource-container` all the time, and only specify `--cgroup-root` when there actually was one to specify.